serverless/networking/serverless-ossm.adoc

```diff
@@ -81,7 +81,7 @@ spec:
 $ oc apply -f <filename>
 ----
 
-. Enable sidecar injection to your Knative Service by adding the `sidecar.istio.io/inject="true"`` annotation to the Service resource:
+. Enable sidecar injection to a service by adding the `sidecar.istio.io/inject="true"` annotation to the service resource:
```

Comment:
Perhaps it would make sense to use `oc annotate ksvc hello sidecar.istio.io/inject="true"` here as well? (same as suggested earlier in this PR)

Reply:
@nak3 any thoughts on replacing the YAML examples with `oc`, or do we need to show both?

@markito might have some thoughts on this. Can we start defining better maybe what we want the user experience to be when documenting a feature, e.g. if it's worth documenting YAML, or just provide `oc` / `kn` CLI, or even maybe just web console docs?

Reply:
@abrennan89 `oc` or `kn` commands cannot add `spec.template.metadata.annotations` so we cannot show `oc` example.

I agree that we should improve the user experience but https://issues.redhat.com/browse/SRVCLI-206 or kn's plugin feature (both are not available yet) will be necessary.

Reply:
Got it, thanks @nak3 ! I'm following the Jira issue now so once this is possible we can doc it using `kn` / `oc` in a follow up. Resolving the comment for this PR.